yarn.lock

```diff
@@ -15433,7 +15426,22 @@ ethereumjs-vm@^2.0.2, ethereumjs-vm@^2.3.4, ethereumjs-vm@^2.6.0:
     rustbn.js "~0.2.0"
     safe-buffer "^5.1.1"
 
-ethers@5.0.32, ethers@^5.0.18:
+ethers@4.0.48:
```

Comment:
two different versions of the same package? Not sure how this even works...

Reply:
`yarn list` shows:

```
├─ @metafam/backend@0.1.0
│  ├─ ethers@5.0.32
├─ @metafam/discord-bot@0.1.0
│  ├─ ethers@4.0.48
├─ @metafam/utils@1.0.0
│  ├─ ethers@5.0.32
├─ @metafam/web@0.1.0
│  ├─ ethers@5.0.32
```
¿So, I'd update the discord bot?

Reply:
Yes. That is what I'd do.

Reply:
Good catch! I actually noticed the same thing on my discord branch after I posted this comment..